app/src/main/res/layout/fragment_recyclerview_row.xml

```diff
@@ -0,0 +1,80 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
Same as above here as well. Let's not hard code width and lengths in this file as well.